cassdegrees/templates/list.html

```diff
@@ -59,7 +59,11 @@
                                         <td>{{ item }}</td>
                                     {% else %}
                                         <td><input title="Select" type="radio" name="id" value="{{ item }}"  /></td>
-                                        <td>(<a href="/api/model/{{ title.lower }}/{{ item }}/">View</a>)</td>
+                                        {% if title.lower == "degree" or title.lower == "subplan" %}
+                                            <td>(<a href="/edit/{{ title.lower }}/{{ item }}/">View</a>)</td>
```

Comment:
Not sure I particularly like this system of a single endpoint for degrees (should be programs) and subplans. They are not identical, and I feel that this will just lead to heaps of if statements in code.

Reply:
Happy to review this at a later date. It should be programs, but we need to go through and refactor the rest of our code to reflect this. At the moment, the List portion of develop generates a "degree" label, not a programs label.

In terms of endpoint, with good use of widgets, and single landing page for editing is viable, but it would be good to have a discussion tomorrow about how much difference there is actually going to be between programs and subplans (I have a feeling once they are fleshed out they will actually be rather similar).

Reply:
We are going to need to have a cleanup of these endpoints at some point *anyway*, so we can have this discussion then, sure,